src/Tools/GenerateDocumentationAndConfigFiles/Program.cs

```diff
@@ -1624,17 +1624,19 @@ static string GetCodeAnalysisTreatWarningsAsErrorsTargetContents()
 ";
             }
 
-            static string GetPackageSpecificContents(string packageName)
-                => packageName switch
-                {
-                    CodeAnalysisAnalyzersPackageName => @"
-  <!-- Target to add all 'EmbeddedResource' files with '.resx' extension as analyzer additional files -->
+            const string AddAllResxFilesAsAdditionalFilesTarget = @"  <!-- Target to add all 'EmbeddedResource' files with '.resx' extension as analyzer additional files -->
   <Target Name=""AddAllResxFilesAsAdditionalFiles"" BeforeTargets=""GenerateMSBuildEditorConfigFileCore;CoreCompile"" Condition=""'@(EmbeddedResource)' != '' AND '$(SkipAddAllResxFilesAsAdditionalFiles)' != 'true'"">
     <ItemGroup>
       <EmbeddedResourceWithResxExtension Include=""@(EmbeddedResource)"" Condition=""'%(Extension)' == '.resx'"" />
       <AdditionalFiles Include=""@(EmbeddedResourceWithResxExtension)"" />
     </ItemGroup>
-  </Target>
+  </Target>";
+
+            static string GetPackageSpecificContents(string packageName)
+                => packageName switch
+                {
+                    CodeAnalysisAnalyzersPackageName => $@"
```

Comment:
Related to @AArnott comment that he needed to add AdditionalFiles himself. Do we need this both for analyzers and resx gen packages? Currently it's only for `Microsoft.CodeAnalysis.Analyzers`.